### ISSUE TITLE
Internationalization 

### ISSUE THREAD

**Issue by 0xmunger** [larvalabs/cryptopunksmarket#4]
This is probably getting a little ahead of where the repo currently is but internationalization is of the site would be huge. Are there plans for this or have you thought about it @megamattron ? I haven't done it personally but I'm happy to look in to it. 

Specifically it appears that Chinese localization would go a long way.

[Cryptopunks google trends](https://trends.google.com/trends/explore?q=cryptopunks)
![image](https://user-images.githubusercontent.com/90461460/147360045-0667288e-1554-4c5b-968e-ad7e3c744a40.png)

**Comment by megamattron** [comment]
Yes that was one of the goals with the frontend rewrite, to allow localization. It's written with vue.js so one of the libraries available for that will probably be the best option.

**Comment by megamattron** [comment]
Parking this here for reference as the likely way to do internationalization: https://kazupon.github.io/vue-i18n/
